src/controllers/utility.controller.ts

```diff
@@ -0,0 +1,36 @@
+import { Context } from "koa";
+import fs from "fs";
+
+export default class UtilityController {
```

Comment:
Let's write a parallel service module to move the response logic there.

Reply:
There isn't much logic to move to a service you have anything specific in mind

Reply:
Even if the logic is simpler, it's better to follow the convention and move it to a service. Future developers can make mistakes and extend functionality here itself.

Reply:
Done